python/ray/autoscaler/cli_logger.py

```diff
@@ -513,6 +513,30 @@ def confirm(self, yes, msg, *args, _abort=False, _default=False, **kwargs):
 
         return res
 
+    def prompt(self, msg: str, *args, **kwargs):
+        """Display a confirmation dialog.
+
+        Args:
+            msg (str): The mesage to display to the user before the prompt.
+        """
+        if self.old_style:
+            return
+
+        complete_str = cf.underlined(msg + ":") + " "
```

Comment:
Consider dropping the `:` here? It might be useful to do stuff like `cli_logger.prompt("How many nodes to launch?")` and we would probably like to see `...launch? 12` instead of the current `...launch?: 12`

The counter-argument to this is that we are not really in the business of writing chat bots, so maybe enforcing `:` is good to encourage utilitarian interfaces :)

Reply:
fixed